operator/controller/src/test/java/io/apicurio/registry/operator/it/ITBase.java

```diff
@@ -54,7 +54,7 @@ public enum OperatorDeployment {
     private static Operator operator;
 
     @BeforeAll
-    public static void before() throws Exception {
+    public static void beforeAll() throws Exception {
```

Comment:
nitpick: the operator codebase is still early and I don't mind those changes, going forward let's try to minimize those unrelated changes, and, maybe pack them in a "chore" separate PR.
Two reasons:
- git history -> is easier to follow the history of changes when they are done in isolation
- review -> it keeps things better scoped

Reply:
no need to process for this, just open a PR changing just this bit and we fast-track it.